ArduPlane/RC_Channel.cpp

```diff
@@ -239,7 +240,8 @@ void RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const AuxSwit
         break;
 
     case AUX_FUNC::FLAP:
-        break; // flap input label, nothing to do
+    case AUX_FUNC::AIRBRAKE:
```

Comment:
Where are we calling `do_aux_function` where `AUX_FUNC::AIRBRAKE`?  That is an *error* in the same way that calling it on `AUX_FUNC::ROLL` would be.  The warning may be entirely justified.

Reply:
I'm not sure what you're getting at. We're using it the same way as Rover's AUX_FUNC::ROLL is. Please look at Plane::AUX_FUNC::FLAP for inspiration.

assignment: https://github.com/ArduPilot/ardupilot/blob/master/Rover/radio.cpp#L21
handler: https://github.com/ArduPilot/ardupilot/blob/master/Rover/RC_Channel.cpp#L46
usage: https://github.com/ArduPilot/ardupilot/blob/master/Rover/mode.cpp#L190

Where AUX_FUNC::AIRBRAKE is used (very similar to AUX_FUNC::FLAP):
assignment: https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/radio.cpp#L45
handler (missing): https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/RC_Channel.cpp#L145
usage: https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/servos.cpp#L291